signac/contrib/project.py

```diff
@@ -520,8 +520,10 @@ def find_job_ids(self, filter=None, doc_filter=None, index=None):
     def find_jobs(self, filter=None, doc_filter=None, index=None):
```

Comment:
We will need to remember to remove the `index` argument here.

Reply:
For what milestone? We can make an issue now and tag it appropriately.

Reply:
For 1.0.0, I just created the issue: https://github.com/glotzerlab/signac/issues/131